modules/javafx.graphics/src/main/java/com/sun/glass/ui/gtk/GtkRobot.java

```diff
@@ -32,8 +33,30 @@
 import com.sun.glass.ui.GlassRobot;
 import com.sun.glass.ui.Screen;
 
+import java.security.AccessController;
+import java.security.PrivilegedAction;
+
+@SuppressWarnings("removal")
```

Comment:
Maybe this can go on the individual elements that use the security manager? To do this, you might need to create a local "tmp" variable and then assign that to `screenshotMethod`.
(either way, don't worry about the files you copied from AWT)

Reply:
Done for all cases.